packages/flutter_goldens/test/flutter_goldens_test.dart

```diff
@@ -34,6 +34,17 @@ const List<int> _kFailPngBytes =
   120, 1, 99, 249, 207, 240, 255, 63, 0, 7, 18, 3, 2, 164, 147, 160, 197, 0,
   0, 0, 0, 73, 69, 78, 68, 174, 66, 96, 130];
 
+void testWithOutput(String name, Future<void> body(), String expectedOutput) async {
```

Comment:
To satisfy the analyzer:
```suggestion
Future<void> testWithOutput(String name, Future<void> body(), String expectedOutput) async {
```